release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -601,6 +601,13 @@ With this release, all `etcd` certificates originate from a new namespace: `open
 
 Manual rotation of signer certificates is still supported by deleting the specific secret and waiting for the status pod rollout to complete.
 
+[id="ocp-17-sigstore-verification_{context}"]
+==== Sigstore signature image verification
+
+With this release, Technology Preview clusters use Sigstore signatures to verify images that were retrieved using a pull spec that references `quay.io/openshift-release-dev/ocp-release`.
```

Comment:
Nit that you could take/leave:
```suggestion
With this release, Technology Preview clusters use Sigstore signatures to verify images that were retrieved using a pull spec that references the `quay.io/openshift-release-dev/ocp-release` image.
```

Reply:
I spoke with @wking and I'll implement the spirit of this feedback, however "the `quay.io/openshift-release-dev/ocp-release` repository" would be more technically accurate here.